recipes/mdsplus/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "mdsplus" %}
+{% set version = "7.7.9" %}
+{% set sha256 = "5b54e4f7bb00afee7ca9ad0fd7f5fda69afe39c817f68c7739939b1415f05008" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/MDSplus/{{ name }}/archive/stable_release-{{ version | replace('.','-') }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - configure.patch.osx  # [osx]
+    - configure.patch.linux  # [linux]
+    - mdsdcl_show_version.c.patch
+    - Makefile.in.patch
+    - mdsdcl_Makefile.in.patch
```

Comment:
What are these patches for? Have they been submitted upstream?

Reply:
The patches are for correcting where to find dependencies (/usr/local -> ${prefix}).  Some may be acceptable upstream.  At least some parts will not be.

Reply:
A comment above the patches would be nice.

Raising an issue or PR to propose them upstream would be great. Ideally we want the developers to integrate fixes so that the burden is removed from the maintainers (yourself included). 😄